### DISCUSSION TITLE
Creation fails

### DISCUSSION THREAD

**Discussion by CircleBoy** [cat-box/aws-foundry-ssl#48] [Q&A]
I get the below error when I try to create the cloudformation. Any ideas?

![image](https://github.com/cat-box/aws-foundry-ssl/assets/16327833/2b4fe415-65a5-4d1f-ae94-dbbe3bb9569e)

**Comment by armandofreire** [comment]
Hey, @CircleBoy , did you manage to get this error fixed? I'm facing the same issue 😢

**Comment by armandofreire** [comment]
Hey @CircleBoy, I think I might have solved it. Replace the FoundryBucket object for this one:

`"FoundryBucket": {
        "Condition": "NoExistingS3",
        "Type": "AWS::S3::Bucket",
        "Properties": {
            "BucketName": {
                "Ref": "S3BucketName"
            },
            "OwnershipControls": {
                "Rules": [
                    {
                        "ObjectOwnership": "ObjectWriter"
                    }
                ]
            },
            "PublicAccessBlockConfiguration": {
                "BlockPublicAcls": false,
                "BlockPublicPolicy": false,
                "IgnorePublicAcls": false,
                "RestrictPublicBuckets": false
            },
            "CorsConfiguration": {
                "CorsRules": [
                    {
                        "AllowedOrigins": [
                            "*"
                        ],
                        "AllowedHeaders": [
                            "*"
                        ],
                        "AllowedMethods": [
                            "GET",
                            "POST",
                            "HEAD"
                        ],
                        "MaxAge": 3000
                    }
                ]
            }
        }
    },`


I extracted this part from @mikehdt (https://github.com/mikehdt/aws-foundry-ssl) JSON. The instance was created with no errors, however I still cant access it. @cat-box Might give us a hint here 😄
